internal/provider/resource_tfe_workspace.go

```diff
@@ -578,8 +591,9 @@ func resourceTFEWorkspaceRead(d *schema.ResourceData, meta interface{}) error {
 		if err != nil {
 			return fmt.Errorf("Error reading auto destroy activity duration: %w", err)
 		}
-
 		d.Set("auto_destroy_activity_duration", v)
+	} else {
+		d.Set("auto_destroy_activity_duration", nil)
```

Comment:
very minor nit, imitating the logic above for other nilable strings (and what we do in the project data source) would probably be better

```go
		var autoDestroyDuration string
		if proj.AutoDestroyActivityDuration.IsSpecified() {
			autoDestroyDuration, err = proj.AutoDestroyActivityDuration.Get()
			if err != nil {
				return diag.Errorf("Error reading auto destroy activity duration: %v", err)
			}
		}
```